client/dashboard/sites/domains/index.tsx

```diff
@@ -51,12 +51,14 @@ function SiteDomains() {
 		fields
 	);
 
+	const redirectTo = `/ciab/sites/${ site.slug }/domains`;
+
```

Comment:
Can't do this (at least not here) since we're not in a ciab-specific component.

Reply:
Good call. I can change it to look at the url instead.